packages/browser-extension/source/localization/localizations.ts

```diff
@@ -0,0 +1,96 @@
+import { SourceFile, factory, ScriptKind, ScriptTarget, createSourceFile, Printer } from "typescript"
+import { TransformTraversalControl, ts } from "ts-morph";
+import { Project } from "ts-morph"
+import sk_SK from "./sk_SK";
+import { LocaleWithPosition, Position } from "./localizationInterfaces";
+import { findElementInCode } from "../tsx/clone";
+import { astFindSource, SourceLineCol } from "../tsx/ast";
+
+
+
+export function getValuesFromLocalizationASTJSON(astLocale: SourceFile | undefined) {
+  let english: string[] = []
+  let slovak: string[] = []
+  let locales: string[] = []
+  let positionsTable: Position[] = []
+  let englishPositionsTable: LocaleWithPosition[] = []
+  astLocale?.forEachChild((child: any) => {
+    child?.expression?.properties?.forEach((property: any) => {
+      console.log("Property", property)
+      locales = [...locales, property.name.text]
+      locales = [...locales, property.initializer.text]
+      positionsTable = [...positionsTable, { text: property.name.text, start: property.name.pos, end: property.name.end },
+      { text: property.initializer.text, start: property.initializer.pos, end: property.initializer.end }]
+      englishPositionsTable = [...englishPositionsTable, { text: property.name.text, position: property.name.pos }]
+      english = [...english, property.name.text]
+      slovak = [...slovak, property.initializer.text]
+    })
+  })
+  console.log("POsitions", positionsTable, "English positions", englishPositionsTable)
+  return {
+    english,
+    slovak,
+    locales,
+    positionsTable,
+    englishPositionsTable
+  }
+}
+
+
+export function saveTableValuesAndParseBack(tableBody: any, positions: LocaleWithPosition[], allPositions: Position[]) {
+  let englishTable: string[] = []
+  let slovakTable: string[] = []
+  let counter = 0
+  for (var r = 0, n = tableBody.rows.length; r < n; r++) {
+    for (var c = 0, m = tableBody.rows[r].cells.length; c < m; c++) {
```

Comment:
Strange data struscutre used - just a simple array. Use objects inside arrays to take advantage of compiler validations.